contracts/BackedAutoFeeTokenImplementation.sol

```diff
@@ -186,14 +213,16 @@ contract BackedAutoFeeTokenImplementation is BackedTokenImplementation {
         public
         view
         virtual
-        returns (uint256 newMultiplier, uint256 periodsPassed)
+        returns (uint256 newMultiplier, uint256 periodsPassed, uint256 newMultiplierNonce)
     {
         periodsPassed = (block.timestamp - lastTimeFeeApplied) / periodLength;
         newMultiplier = multiplier;
+        newMultiplierNonce = multiplierNonce;
         if (feePerPeriod > 0) {
             for (uint256 index = 0; index < periodsPassed; index++) {
                 newMultiplier = (newMultiplier * (1e18 - feePerPeriod)) / 1e18;
             }
+            newMultiplierNonce += periodsPassed;

```

Comment:
should be outside of the loop!

Reply:
it is outside of the loop, do you mean outside of the if? should we increase nonce even if we don't change multiplier?

Reply:
sorry I don't know what I was thinking :)

Reply:
no worries, I think you might have had outdated changes from yeserday